picoharpApp/src/picopeaks.h

```diff
@@ -1,10 +1,10 @@
 #ifndef __PICOPEAKS_H__
 #define __PICOPEAKS_H__
 
-#include <phdefin.h>
+#include "ph330defin.h"
 
 #define BLOCK 0
-
+#define HISTCHAN DFLTHISTLEN // 65536 max is 524288 now!
```

Comment:
PicoHarp version specific, evidently.

Reply:
HISTCHAN doesn't exists anymore in ph330defin.h, this only wraps the successor to the old define